src/ui/public/kuery/lib/functions/not.js

```diff
@@ -0,0 +1,38 @@
+import * as ast from '../ast';
+import { nodeTypes } from '../node_types';
+
+export function buildNodeParams(child, serializeStyle = 'operator') {
+  return {
+    arguments: [child],
+    serializeStyle
+  };
+}
+
+export function toElasticsearchQuery(node, indexPattern) {
+  let [ argument ] = node.arguments;
+  if (argument.type === 'literal') {
+    argument = nodeTypes.function.buildNode('is', '*', argument.value);
```

Comment:
Similar to the other case, I feel this should have happened earlier (probably in `buildNodeParams`).